stl/inc/random

```diff
@@ -1005,16 +999,28 @@ public:
 };
 
 #if _HAS_TR1_NAMESPACE
+_CONSTEVAL double _Cx_ldexp_one(const int _Exp) noexcept {

```

Comment:
I have to say I absolutely hate this name; is it at all possible to call it something like `_Cx_exp2` or something? Make it obvious that this is `2^x`?

Reply:
Changed. I'm sorry for my laziness which drove me to reuse `ldexp` in this internal name.

Reply:
thanks!